date_module.js

```diff
@@ -0,0 +1,88 @@
+//setdate
+// set a variable for the date using built in js date function
+// setDate is a method that allows the value of the date variable to be changed by user input
+// user enters a new date which is saved to variable
+
+var myObj = ( function iife() {
+  var myDate = new Date();
+
+  function setDate() {
```

Comment:
This method should take one parameter that you use to instantiate your internal date instance with if they've provided either a millisecond value or a date instance (see sample usage for the method on the Module_2 README.